src/libre/ast.c

```diff
@@ -893,3 +893,22 @@ ast_make_expr_named(struct ast_expr_pool **poolp, enum re_flags re_flags, const
 	return NULL;
 }
 
+const char *
+ast_node_type_name(enum ast_expr_type t)
+{
+	switch (t) {
+	default:
+		return "<match fail>";
```

Comment:
convention is for `return "?";` in this repo

Reply:
Fixed in 676e61d0.